examples/fabric/connection/example_fcr_to_azure/go/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21
 toolchain go1.22.6
 
 require (
-	github.com/equinix/pulumi-equinix/sdk 0.17.0
+	github.com/equinix/pulumi-equinix/sdk latest
```

Comment:
Oh, I like this!  We'll need to be careful about not committing changes after running `go mod tidy`, but this is a nice, simple solution to the issue we're having.